src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap");
+$main-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)